docs/Oracle Jobs/Job Types/direct_request.md

```diff
@@ -96,3 +99,86 @@ encode_tx       [type=ethabiencode
 
 submit_tx  [type=ethtx to="0x613a38AC1659769640aaE063C651F48E0250454C" data="$(encode_tx)"]
 ```
+
+**Multi word example**
+
+Say a user wants to obtain the ETH price quoted against 3 different fiat currencies. If they were only using single word DR jobs, it would require 3 different requests. To make that more efficient they use multi word responses as shown below to do it all in a single request.
```

Comment:
```suggestion
Assume that a user wants to obtain the ETH price quoted against three different fiat currencies. If they use only a single-word DR job, it would require three different requests. To make that more efficient, they can use multi word responses to do it all in a single request as shown in the following example:
```